networking/metallb/metallb-frr-k8s.adoc

```diff
@@ -0,0 +1,27 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="metallb-configure-frr-k8s"]
+= Configuring integration of MetalLB and FRR-K8s
+include::_attributes/common-attributes.adoc[]
+:context: configure-metallb-frr-k8s
+
+toc::[]
+
+:FeatureName: The `FRRConfiguration` custom resource
```

Comment:
Assuming this is the entry point, we must add somewhere the warning that the feature is Tech Preview

Reply:
the `include` statement on line 10 includes the standard "Tech Preview" snippet. It's not visible here but you can see it in the [docs preview](https://75492--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/metallb/metallb-frr-k8s).